apps/imgviewer.c

```diff
@@ -47,22 +49,36 @@ int height = 300;
 sprite_t img = {0};
 
 #define APPLICATION_TITLE "Image Viewer"
+#define IMGVIEWER_VERSION   "1.4.0" 
+#define IMGVIEWER_COPYRIGHT "Copyright 2012-2019 K. Lange <\033[3mklange@toaruos.org\033[23m>"
 
+#define DEBUG 1
+
+/**
+ * Show help text for -?
+ */
 void usage(char * argv[]) {
+#define _s "\033[3m"
+#define _e "\033[0m\n"
 	printf(
 			"Image Viewer - Shows images.\n"
 			"\n"
-			"usage: %s \033[3mimage\033[0m\n"
+			"usage: %s [file image]\n"			
```

Comment:
Brackets indicate optional arguments and "file image" doesn't make sense. "image file" would, but using two words for an argument indicator like this is poor style.